spi_nand_flash/idf_component.yml

```
@@ -0,0 +1,11 @@
+version: "0.1.0"
+description: Driver for accessing SPI NAND Flash
+url: https://github.com/espressif/idf-extra-components/tree/master/spi_nand_flash
+issues: https://github.com/espressif/idf-extra-components/issues
+repository: https://github.com/espressif/idf-extra-components.git
+documentation: https://github.com/espressif/idf-extra-components/tree/main/spi_nand_flash/README.md
+dependencies:
+  idf: ">=5.0"
+  dhara:
+    version: "0.1.0"
```

Do we perhaps want to automatically update the dependency if there is a bugfix release?

```suggestion
    version: "0.1.*"
```